doc/howto/images_manage.md

```diff
@@ -29,9 +57,32 @@ To filter for properties other than alias or fingerprint, specify the filter in
 For example:
 
     lxc image list ubuntu: 22.04 architecture=x86_64
+```
+```{group-tab} API
+You can {ref}`filter <rest-api-filtering>` the images that are displayed by any of their fields.
+
+For example, to show all Ubuntu images, or all images for version 22.04:
+
+    lxc query --request GET /1.0/images?filter=properties.os+eq+ubuntu
+    lxc query --request GET /1.0/images?filter=properties.version+eq+22.04
+
+You can specify several filters as well.
+For example, to show all Arm 64-bit images for virtual machines, enter the following command:
+
+    lxc query --request GET /1.0/images?filter=architecture+eq+arm64+and+type+eq+virtual-machine
+
+You can also use a regular expression:
+
+    lxc query --request GET /1.0/images?filter=fingerprint+eq+be25.*
```

Comment:
I'd suggest when including asterisk (or other shell globbing) characters that you quote the arguments so the shell doesn't potentially replace them with file names from the current directory.